tools/options.py

```diff
@@ -121,3 +121,12 @@ def extract_profile(parser, options, toolchain, fallback="develop"):
                                 " supported by profile {}").format(toolchain,
                                                                    filename))
     return profile
+
+def not_disabled(parser, mcu):
```

Comment:
this looks like generic function but semes like only for cortex-a, should it be an arugment and related to MCU (mcu in the name?) . Why is it negative by default - not_disabled?

should it be ``is_core_disabled(parser, core, mcu) ? `` However this would make more work to get the url link work with the release one. If no, at least ``is_cortex_a_disabled`` or something ?